SingularityUI/app/components/groupDetail/GroupDetail.jsx

```diff
@@ -1,5 +1,6 @@
 import React, {PropTypes, Component} from 'react';
 import { connect } from 'react-redux';
+import { LinkContainer } from 'react-router-bootstrap';
```

Comment:
What's the reasoning behind using `LinkContainer` over the standard react-router `Link` or an onclick handler?

Reply:
`Link` doesn't play nice with bootstrap components. I also wanted the URL to change when a user clicked on a request ID which is why I went with `LinkContainer`. Would it have been better to have an onClick handler and wrap the component using `withRouter`?

Reply:
`Link` doesn't play nice with bootstrap components. I also wanted the URL to change when a user clicked on a request ID which is why I went with `LinkContainer`. Would it have been better to have an onClick handler and wrap the component using `withRouter`?

Reply:
If we were already using `react-router-bootstrap` I'd say this is the best solution, however, I'm hesitant to add another dependency for this single use case since we already have so many. For that reason I'd say go with handling it via `onClick` and `withRouter`.